docs/gravitino-server-config.md

```diff
@@ -43,14 +43,20 @@ You can also specify filter parameters by setting configuration entries of the f
-| `gravitino.entity.store.kv`                       | Detailed implementation of KV storage. `RocksDB` storage is currently supported, and the implementation is `RocksDBKvBackend`.                          | `RocksDBKvBackend`               | No       | 0.1.0         |
-| `gravitino.entity.store.kv.rocksdbPath`           | Directory path of `RocksDBKvBackend`.                                                                                                       | `${GRAVITINO_HOME}/data/rocksdb` | No       | 0.1.0         |
-| `graivitino.entity.serde`                         | The serialization/deserialization class used to support entity storage. `proto' is currently supported.                                     | `proto`                          | No       | 0.1.0         |
-| `gravitino.entity.store.maxTransactionSkewTimeMs` | The maximum skew time of transactions in milliseconds.                                                                                      | `2000`                           | No       | 0.3.0         |
-| `gravitino.entity.store.kv.deleteAfterTimeMs`     | The maximum time in milliseconds that deleted and old-version data is kept. Set to at least 10 minutes and no longer than 30 days. | `604800000`(7 days)              | No       | 0.3.0         |
+| Configuration item                                | Description                                                                                                                                                               | Default value                    | Required | Since version |
+|---------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|----------|---------------|
+| `gravitino.entity.store`                          | Which storage implementation to use. Key-value pair storage and Relational storage is currently supported, the default value is `kv`, the optional value is `relational`. | `kv`                             | No       | 0.1.0         |
+| `gravitino.entity.store.kv`                       | Detailed implementation of KV storage. `RocksDB` storage is currently supported, and the implementation is `RocksDBKvBackend`.                                            | `RocksDBKvBackend`               | No       | 0.1.0         |
+| `gravitino.entity.store.relational`               | Detailed implementation of Relational storage. `MySQL` storage is currently supported, and the implementation is `JDBCBackend`.                                           | `JDBCBackend`                    | No       | 0.5.0         |
```

Comment:
Can you provide some descriptions of the pre-steps like `You should create tables by execute mysql_init.sql` or something else?